src/Elcodi/Bundle/GeoBundle/README.md

```diff
@@ -44,7 +44,14 @@ bundle, you'll find info on this [symfony documentation page][4]
 In a few words, you can use [Composer] to install the bundle getting the package
 from
 [elcodi/geo-bundle packagist](https://packagist.org/packages/elcodi/geo-bundle)
-by just adding a line in your composer.json
+by just executing the following line
+
+``` bash
+$ composer require "elcodi/geo-bundle:~0.5.*"
+```
+
+You can also do it manually by adding a line in your ``composer.json`` file
```

Comment:
unnecessary double backticks here: they are used in reStructuredText doc, but this is Markdown